deploy/openshift/install-openshift.sh

```diff
@@ -0,0 +1,23 @@
+#! /bin/sh
+#
+#
+# Install Kiali 
+
+
+
+JAEGER_URL="http://jaeger-query-istio-system.127.0.0.1.nip.io"
+GRAFANA_URL="http://grafana-istio-system.127.0.0.1.nip.io"
+VERSION_LABEL="v0.9.0"
+IMAGE_PULL_POLICY_TOKEN="imagePullPolicy: Always"
+NAMESPACE="istio-system"
```

Comment:
We need to support users that want to override these by setting the env vars. So all of these should be of the form:

ENV_VAR_NAME=${ENV_VAR_NAME:-defaultValue}

For example, these should be:

```
JAEGER_URL="${JAEGER_URL:-http://jaeger-query-istio-system.127.0.0.1.nip.io}"
GRAFANA_URL="${GRAFANA_URL:-http://grafana-istio-system.127.0.0.1.nip.io}"
VERSION_LABEL="${VERSION_LABEL:-v0.9.0}"
IMAGE_PULL_POLICY_TOKEN="${IMAGE_PULL_POLICY_TOKEN:-imagePullPolicy: Always}"
NAMESPACE="${NAMESPACE:-istio-system}"
DOCKER_NAME="${DOCKER_NAME:-kiali/kiali}"
VERBOSE_MODE="${VERBOSE_MODE:-4}
```

Now a person can run something like:

```
JAEGER_URL="http://my-jaeger" VERBOSE_MODE="3" VERSION_LABEL="v0.9.1" install-openshift.sh
```